packages/screenshots/Classes/Configuration/Configuration.php

```diff
@@ -270,6 +297,64 @@ public function createBasicConfig(): void
                             ['action' => 'click', 'link' => "List"],
                             ['action' => 'makeScreenshotOfFullPage', 'fileName' => "IntroductionDashboardFullPage"],
                         ],
+                        "startpage" => [
+                            [
+                                "action" => "setScreenshotsImagePath",
+                                "path" => "Images/AutomaticScreenshots/IntroductionPackage"
+                            ],
+                            [
+                                "action" => "setScreenshotsRstPath",
+                                "path" => "Images/AutomaticScreenshots/IntroductionPackage"
+                            ],
+                            [
+                                "action" => "resizeWindow",
+                                "width" => 1024,
+                                "height" => 512
+                            ],
+                            [
+                                "action" => "see",
+                                "text" => "Page"
+                            ],
+                            [
+                                "action" => "click",
+                                "link" => "Page"
+                            ],
+                            [
+                                "action" => "openPageTreePath",
+                                "path" => ["Congratulations"]
+                            ],
+                            ["action" => "switchToContentFrame"],
+                            [
+                                "action" => "waitForText",
+                                "text" => "Congratulations",
+                                "timeout" => 5
+                            ],
+                            [
+                                "action" => "makeScreenshotOfElement",
```

Comment:
Prefer `makeScreenshotOfWindow` over `makeScreenshotOfElement` if no selector is specified ..

Reply:
Why? Make screenshotOfWindow only seems to make sense when you want to catchOverlays or the such